spec/models/miq_schedule_spec.rb

```diff
@@ -47,6 +56,12 @@
             expect(miq_schedule_array['filter_resource_name']).to be_nil
           end.not_to raise_error
         end
+
+        it "raises on import" do
```

Comment:
Sorry, but it's not obvious to me... What's wrong with this?

Reply:
https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_schedule.rb#L8 because schedules are invalid without resource_types, so we shouldn't be able to import them if they don't exist

Reply:
So, in this case, we have an invalid schedule already in the database, export it, then expect a failure when trying to import it?

Reply:
😆 Yuri told me to write import specs that used what we exported, so absolutely. What could go wrong?

Reply:
export/import is not atomic operation, deletion may happened after export